app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    source_file = parts[1]
```

Comment:
There can use comprehension to get params from array, would be like this "_, source_file, target_file = parts"